### PR TITLE
Walking skeleton

### PR DESCRIPTION
**Pull request by mrfishball**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    "node/process-exit-as-throw": "error",
+
+    "promise/param-names": "error",
+
+    "standard/array-bracket-even-spacing": ["error", "either"],
+    "standard/computed-property-even-spacing": ["error", "even"],
+    "standard/no-callback-literal": "error",
+    "standard/object-curly-even-spacing": ["error", "either"]
+  }
+}
```
Comment (cmvandrevala) [inline .eslintrc:206]: You don't have to include all of the eslint rules - just the ones that make sense for your project and your team. I am assuming these are the defaults, so they are fine for now.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+### Prerequisites
+
+You're going to need:
+
+ - **Node.js/NPM, version 11.1.0 or newer**
+ - **Any modern browser, latest stable version**
+
+### Getting Set Up
+
+1. Fork this repository on GitHub.
```
Comment (cmvandrevala) [inline README.md]: Why do I need to fork this repo? Why not just clone it directly?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "webtttclient",
+  "version": "1.0.0",
+  "description": "A Tic Tac Toe client that runs in the web browser",
+  "main": "index.js",
+  "scripts": {
+    "start": "node server.js",
```
Comment (cmvandrevala) [inline package.json:7]: I like your scripts in package.json

File: test/index_test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+describe('fetchBoard', () => {
+  it('fetches from the TicTacToeRubyRails API', () => {
+    spyOn(window, 'fetch').and.callThrough()
+    fetchBoard()
+
+    expect(window.fetch).toHaveBeenCalledWith('https://live-ttt-rubyrails-api.herokuapp.com/v1/game')
```
Comment (cmvandrevala) [inline test/index_test.js]: Why is this production url hard-coded in your tests?

**Review by cmvandrevala — CHANGES_REQUESTED** [review]
I have a few small nitpicks/questions about this, but otherwise I really like this walking skeleton. Great job!

File: .eslintrc (hunk 1 above)
Comment (mrfishball) [reply .eslintrc:206]: I want to have some rules in place to make sure the syntax is as readable and consistent as possible but, I didn't want to spend too much time on configuring the linter so I found a .eslintrc on GitHub and just went with it.

File: README.md (hunk 2 above)
Comment (mrfishball) [reply README.md]: I suppose clone makes more sense. will change it to clone instead

File: package.json (hunk 3 above)
Comment (mrfishball) [reply package.json:7]: :)

File: test/index_test.js (hunk 4 above)
Comment (mrfishball) [reply test/index_test.js]: will fix that

**Review by ericmeyer — APPROVED** [review]

**Pull request merged**